src/content/app/about/components/about-menu/AboutMenu.tsx

```diff
@@ -48,33 +51,53 @@ export const TopMenu = (props: Props) => {
 export const SideMenu = (props: Props) => {
   const { menu, currentUrl } = props;
 
-  const menuItem = menu.items.find(
+  const topLevelMenuItem = menu.items.find(
     (menuItem) =>
       menuItem.type === 'collection' &&
-      menuItem.items.find((item) => item.url === currentUrl)
-  );
+      (menuItem.url === currentUrl ||
+        menuItem.items.some((item) => item.url === currentUrl))
+  ) as MenuCollectionItem;
 
-  if (!menuItem || menuItem.type !== 'collection') {
-    // this shouldn't happen
+  if (!topLevelMenuItem) {
     return null;
   }
 
   return (
     <>
-      {menuItem.items.map((item, index) => {
-        const linkClasses = classNames(styles.sideMenuLink, {
-          [styles.activeLink]: item.url === currentUrl
-        });
-        return (
-          <Link className={linkClasses} to={item.url as string} key={index}>
-            {item.name}
-          </Link>
-        );
-      })}
+      {topLevelMenuItem.url &&
+        buildSideMenuLink(
+          topLevelMenuItem as { name: string; url: string },
+          currentUrl,
+          -1
+        )}
+      {topLevelMenuItem.items.map((item, index) =>
+        buildSideMenuLink(
+          item as { name: string; url: string },
+          currentUrl,
+          index
+        )
+      )}
     </>
   );
 };
 
+const buildSideMenuLink = (
+  menuItem: { name: string; url: string },
```

Comment:
Isn't `Pick<MenuCollectionItem, 'name' | 'url'>` better here?

Reply:
The whole file is gone now :-)